route/build/src/mpi_process.f90

```diff
@@ -464,6 +506,13 @@ subroutine comm_ntopo_data(pid,                & ! input: proc id
        structNTOPO_main(nRch_mainstem+ix)%var(ixNTOPO%downSegId)%dat(1) = structNTOPO(ixx)%var(ixNTOPO%downSegId)%dat(1)
        structSEG_main  (nRch_mainstem+ix)%var(ixSEG%length)%dat(1)      = structSEG(ixx)%var(ixSEG%length)%dat(1)
        structSEG_main  (nRch_mainstem+ix)%var(ixSEG%slope)%dat(1)       = structSEG(ixx)%var(ixSEG%slope)%dat(1)
+       structNTOPO_main(nRch_mainstem+ix)%var(ixNTOPO%islake)%dat(1)    = structNTOPO(ixx)%var(ixNTOPO%islake)%dat(1)
```

Comment:
does this look ok? or should be structSEG instead of structNTOPO_main?

Reply:
structNTOPO holds integer type and structSEG holds real type. islake is integer, so this looks ok

Reply:
thank you, yes, the issues was with distributing islake instead that is fixed now... so the code works and yield results as it should be... still need few adjustment to incorporate islake_sim flag into the code so it runs is lakes are set as off.